### ISSUE TITLE
Incorrect opencv dll file names.

### ISSUE THREAD

**Issue by vyacheslav-lonschakov** [peterbraden/node-opencv#320]
I have a opencv-2.4.11 and all lib file names contains '2411' not '249':

`Libs: ${libdir}/opencv_calib3d249 ${libdir}/opencv_contrib249 ${libdir}/opencv_core249 ${libdir}/opencv_features2d249 ${libdir}/opencv_flann249 ${libdir}/opencv_gpu249 ${libdir}/opencv_highgui249 ${libdir}/opencv_imgproc249 ${libdir}/opencv_legacy249 ${libdir}/opencv_ml249 ${libdir}/opencv_nonfree249 ${libdir}/opencv_objdetect249 ${libdir}/opencv_ocl249 ${libdir}/opencv_photo249 ${libdir}/opencv_stitching249 ${libdir}/opencv_superres249 ${libdir}/opencv_ts249 ${libdir}/opencv_video249 ${libdir}/opencv_videostab249`
https://github.com/peterbraden/node-opencv/blob/master/utils/opencv_x64.pc#L13

In the description was not written that it was for a single version.

**Comment by danschultzer** [comment]
What's the issue? I can get `node-opencv` to work with any version of `opencv` I've installed.

**Comment by vyacheslav-lonschakov** [comment]
As i remember, it started to work only after replacement from '249' to '2411' in opencv_x64.pc file (in windows os).

**Comment by danschultzer** [comment]
> As i remember, it started to work only after replacement from '249' to '2411' in opencv_x64.pc file (in windows os).

I've seen that solution in another issue. It looks like [new build script](https://github.com/peterbraden/node-opencv/blob/master/utils/find-opencv.js) [was added](https://github.com/peterbraden/node-opencv/commit/9561b4f71074b69e2a5b475e4e069c57c95fc2b1) for Windows back in January. I will close this as it looks to be resolved with that commit.
